MM/5.10.0/config/media_manager_5/labels/Settings.dcl

```diff
@@ -7145,6 +7163,198 @@ resource configservice_label settings_connector_settings_use_dam_generated_thumb
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'URL\'en til backend-tjenesten i Azure.'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label settings_connector_settings_sales_force_azure_code_title {
+  key = 'SETTINGS_CONNECTOR_SETTINGS_SALES_FORCE_AZURE_CODE_TITLE'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Azure Backend Code'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Azure backend-kode'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label settings_connector_settings_sales_force_azure_code_description {
+  key = 'SETTINGS_CONNECTOR_SETTINGS_SALES_FORCE_AZURE_CODE_DESCRIPTION'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The Code for the backend service in Azure.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Koden til backend-tjenesten i Azure.'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label settings_connector_settings_sales_force_client_id_title {
+  key = 'SETTINGS_CONNECTOR_SETTINGS_SALES_FORCE_CLIENT_ID_TITLE'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'SalesForce Client ID'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'SalesForce klient ID'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label settings_connector_settings_sales_force_client_id_description {
+  key = 'SETTINGS_CONNECTOR_SETTINGS_SALES_FORCE_CLIENT_ID_DESCRIPTION'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The unique identifier assigned to your application within the Salesforce environment, enabling secure access to Salesforce APIs and resources.'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Den unikke identifikator tildelt din ansøgning inden for Salesforce-miljøet, hvilket muliggør sikker adgang til Salesforce API\'er og ressourcer.'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label settings_connector_settings_sales_force_client_secret_title {
+  key = 'SETTINGS_CONNECTOR_SETTINGS_SALES_FORCE_CLIENT_SECRET_TITLE'
+  group = 'Settings'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'SalesForce Client Secret'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'SalesForce klienthemmelighed'
```

Comment:
```suggestion
      default_translation = 'Salesforce client secret'
```

Reply:
I think translating this would make it harder to understand

Reply:
changed